KsApi/models/graphql/adapters/GraphAPI.BlockUserInput+BlockUserInput.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
```

Comment:
For what it's worth - I'm happy to follow the pattern for now, but considering how simple this input is, I'm not entirely sure it's worth having this extra boilerplate for the adapter + adapter tests. Just something to consider in the back of our minds as we move forward with other kinds of API refactoring in the future.